api/src/main/java/io/opentelemetry/spi/TaggerProvider.java

```diff
@@ -14,25 +14,23 @@
  * limitations under the License.
  */
 
-package io.opentelemetry.tags;
+package io.opentelemetry.spi;
+
+import io.opentelemetry.tags.Tagger;
 
 /**
- * Class to access the global {@link Tagger}.
- *
- * @since 0.1.0
+ * TaggerProvider is a service provider for {@link Tagger}. Implementation should register
+ * implementation in <code>META-INF/services/io.opentelemetry.spi.TaggerProvider</code>. <br>
+ * <br>
+ * A specific implementation can be specified by a system property {@code
+ * io.opentelemetry.spi.TaggerProvider} with value of fully qualified class name.
  */
-public final class Tags {
-  private static final Tagger TAGGER = NoopTags.newNoopTagger();
-
-  private Tags() {}
+public interface TaggerProvider {
 
   /**
-   * Returns the default {@code Tagger}.
+   * Creates a new tagger instance.
    *
-   * @return the default {@code Tagger}.
-   * @since 0.1.0
```

Comment:
We should probably keep the since tag.

Reply:
Git recognized this as rename, but it is not a rename.

`Tags.java` has been completely removed

Reply:
I meant to have a since tag for every public method.

Reply:
I will add it in the following PR as it needs to be added in a couple of places in SPI and global registry.